test/routes/routes.test.js

```diff
@@ -31,10 +39,16 @@ const isOnIgnoreList = (endpoint, method) => {
 };
 
 const acceptedResults = (status, endpoint, method) => {
+	if (isOnWhitelist(endpoint)) {
+		let whitelistResult = checkWhitelist(endpoint, method, status);
+		if( typeof(whitelistResult) !== 'undefined' ) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [There should be no space after this paren.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5673434)